### ISSUE TITLE
Multiple Instances

### ISSUE THREAD

**Issue by jcdmacleod** [kilnfi/cosmos-validator-watcher#59]
I believe I am correct in that an instance of the cosmos-validator-watcher is required per monitored chain and that the exposed http port would need to be unique so that prometheus can hit each instance?

**Comment by MattKetmo** [comment]
Yes, exactly. We run as many instances of `cosmos-validator-watcher` as we are active in different chains.
And since we're running it in containers, there is no port overlap, but indeed it would require to change the `--http-addr` if running several on the same host.
